src/graph_compute/bsp_scheduling.h

```diff
@@ -14,14 +14,18 @@ static void init_scheduling(vertex_t * const nodes, const vid_t cntNodes,
   // no-op
 }
 
-static void execute_round(const int round, vertex_t * const nodes, const vid_t cntNodes,
+static void execute_round(const int numRounds, vertex_t * const nodes,
```

Comment:
Not merge, rebase -- very different. If you've made a lot of progress since the latest commit in this PR, then update this PR with all of your branch's changes -- ideally with a rebase but a merge should also work.